src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -903,19 +893,18 @@ void ShortcutManager::RegisterDefaultShortcuts()
             gDoSingleUpdate = true;
         }
     });
-    RegisterShortcut(ShortcutId::DebugTogglePaintDebugWindow, STR_SHORTCUT_DEBUG_PAINT_TOGGLE, []() {
-        if (!(gScreenFlags & SCREEN_FLAGS_TITLE_DEMO))
-        {
-            auto window = window_find_by_class(WC_DEBUG_PAINT);
-            if (window != nullptr)
+    RegisterShortcut(ShortcutId::DebugTogglePaintDebugWindow, STR_SHORTCUT_DEBUG_PAINT_TOGGLE,[]() {
+            if (!(gScreenFlags & SCREEN_FLAGS_TITLE_DEMO))
             {
-                window_close(window);
+                auto window = window_find_by_class(WC_DEBUG_PAINT);
+                if (window != nullptr)
+                {
+                    window_close(window);
+                }
+                else
+                {
+                    context_open_window(WC_DEBUG_PAINT);
+                }
             }
-            else
-            {
-                context_open_window(WC_DEBUG_PAINT);
-            }
-        }
     });
-    // clang-format on
```

Comment:
This line is removed, but the one turning it off at the start is not.

Reply:
This still needs reverting, and the line below needs to be removed, otherwise the scope of these on/off comments do not match.